src/host/main.cpp

```diff
@@ -457,6 +457,7 @@ int main(int argc, char** argv)
   }
 
   const auto cli_config = app.config_to_str(true, false);
+  LOG_INFO_FMT("Version: {}", ccf::ccf_version);
```

Comment:
This is the only change to the shipped CCF binaries, which I believe is harmless but useful for operators when performing code upgrades.